doc/content/articles/miniweb.article

```diff
@@ -0,0 +1,177 @@
+miniweb
+
+The minimega authors
+26 Sept 2017
+
+* Introduction
+
+`miniweb` is a webserver for minimega based on the `web` API that existed until
```

Comment:
I think it would be good to mention why we split miniweb out of the minimega binary... this sentence confuses me a little.

Has all the documentation been updated to reflect that we now use miniweb instead of web?